### DISCUSSION TITLE
[SUGGESTION] Double Pocket prefab door.

### DISCUSSION THREAD

**Discussion by Servomotor** [tsunamayo/Starship-EVO#4017] [Suggestions]
May we get the door as shown in the pictures. This would help to use a larger door (3x3 shown) without having to worry about the door opening to the outside of the ship or building. It would be much easier to manage the double sliding door in builds. 

Thank you.

![we have, would like to have](https://user-images.githubusercontent.com/65745916/107853476-f6c05f80-6ddb-11eb-85eb-52bb78add30f.jpg)

**Comment by ZachZent** [comment]
Also could the doors just become invisible when outside the door frame? They are already incorporeal, they just need to disappear

**Comment by SnowyBirdot** [comment]
I second Zach's comment. It would honestly be quite handy if a bit odd.

**Comment by tsunamayo** [comment]
Yep it makes sense! Thanks

**Comment by Servomotor** [comment]
> Also could the doors just become invisible when outside the door frame? They are already incorporeal, they just need to disappear

My issue for not wanting to see the doors disappear is multi-fold. 

1. If the doors just went *POOF* then some folks might build a door butted up against a brick wall. A wall that would not be thick enough for the door to open into if this was taking place in the real world. This may sound picky, but this can ruin immersion. The door having doors that actually open ensures that builds will "feel right". 

2. I have considered using the doors as blockades or barrier popups for a police station. They would be below 1 surface block and would be used as a security device. Making the doors disappear completely does away with the odd ball ideas someone could come up with, ingenious ways to use the doors other than doors.

**Comment by SnowyBirdot** [comment]
> My issue for not wanting to see the doors disappear is multi-fold.
> 
> 1. If the doors just went _POOF_ then some folks might build a door butted up against a brick wall. A wall that would not be thick enough for the door to open into if this was taking place in the real world. This may sound picky, but this can ruin immersion. The door having doors that actually open ensures that builds will "feel right".
> 2. I have considered using the doors as blockades or barrier popups for a police station. They would be below 1 surface block and would be used as a security device. Making the doors disappear completely does away with the odd ball ideas someone could come up with, ingenious ways to use the doors other than doors.


You know what Servo, you do have a point about the disappearing doors. I feel like a build feeling off could be fixed by just considering your needs for the build, but yeah that would remove some creative potential there as you say.

**Comment by Servomotor** [comment]
> > My issue for not wanting to see the doors disappear is multi-fold.
> > 
> > 1. If the doors just went _POOF_ then some folks might build a door butted up against a brick wall. A wall that would not be thick enough for the door to open into if this was taking place in the real world. This may sound picky, but this can ruin immersion. The door having doors that actually open ensures that builds will "feel right".
> > 2. I have considered using the doors as blockades or barrier popups for a police station. They would be below 1 surface block and would be used as a security device. Making the doors disappear completely does away with the odd ball ideas someone could come up with, ingenious ways to use the doors other than doors.
> 
> You know what Servo, you do have a point about the disappearing doors. I feel like a build feeling off could be fixed by just considering your needs for the build, but yeah that would remove some creative potential there as you say.

Ya the way I see it is that the door opening is a visual reminder to check that the wall it's opening into is thick enough or not. A player could consider their needs as you suggest achieving the same result, but ensuring the door has a physical appearance when it's opened can be used as s visual indicator for the builder to know if the door is "realistic" or not. 

One of the things I absolutely hate about Elder Scrolls Online is their architecture. Buildings work as if they are the Tardis. You can see a fireplace on the outside, go inside and instead you see two windows.  Go to the cats paw bank or whatever its name is in Rimmen and the exterior is smaller than the interior. I have called this out to them for years and they still, to this day refuse to match the interior walls to the exterior (most likely different artists working on the interior instances and exteriors). 

Obviously that specific issue can not be done in StEvo because our interiors are not instanced. But ensuring things like pocket doors maintaining their physical appearance when open will help the builder/artist/designer not make those mistakes.
